tests/AdaptiveExpressions.Tests/BadExpressionTests.cs

```diff
@@ -172,6 +172,10 @@ public class BadExpressionTests
             #endregion
             
             #region Date and time function test
+            Test("hasFullDate(12345)"), // should hava string or TimexProperty parameter
```

Comment:
>hava [](start = 50, length = 4)

have a

Reply:
fixed

Reply:
Still have lots of "hava" in this file that should be "have"

---
In reply to: [414243017](https://github.com/microsoft/botbuilder-dotnet/pull/3780#discussion_r414243017) [](ancestors = 414243017)